staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -167,17 +167,19 @@ func LogRequestPatch(ae *audit.Event, patch []byte) {
 	}
 }
 
-// LogResponseObject fills in the response object into an audit event. The passed runtime.Object
+// LogResponseStatusAndObject fills in the response status and response object into an audit event. The passed runtime.Object
```

Comment:
I would revert the renaming here, IMO it's more confusing now. The status is only set if the object is a `Status` object.